core/services/remote_access_services.py

```diff
@@ -121,8 +121,10 @@ def store_trained_classifier_model(job_result_dict):
     if 'classifier_data' not in job_result_dict:
         raise Exception('job_result_dict must contain \'classifier_data\'.')
 
-    payload = job_result_dict
-    payload['vm_id'] = _get_vm_id()
+    payload = {
+        'message': job_result_dict,
+        'vm_id': _get_vm_id()
+    }
     signature = generate_signature(payload)
     payload['signature'] = signature
```

Comment:
Hm. I thought we wanted the signature to be alongside the payload dict, not in it?